src/main/java/org/jabref/gui/entryeditor/citationrelationtab/RelatedEntriesComponent.java

```diff
@@ -0,0 +1,237 @@
+import org.jabref.preferences.PreferencesService;
+
+import com.tobiasdiez.easybind.EasyBind;
+import org.controlsfx.control.CheckListView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RelatedEntriesComponent extends VBox {
+    private static final Logger LOGGER = LoggerFactory.getLogger(RelatedEntriesComponent.class);
+    private final LibraryTab libraryTab;
+    private final BibEntry pivotEntry;
+    private final RelatedEntriesComponentConfig config;
+    private final RelatedEntriesComponentViewModel viewModel;
+    private Button refreshButton;
+    private Button cancelButton;
+    private ProgressIndicator progressIndicator;
+    private Button importEntriesButton;
+    private CheckListView<CitationRelationItem> relatedEntriesListView;
+
+    public RelatedEntriesComponent(BibEntry pivotEntry, RelatedEntriesRepository repository, LibraryTab libraryTab,
+                                   RelatedEntriesComponentConfig config, DialogService dialogService,
+                                   BibDatabaseContext databaseContext, UndoManager undoManager,
+                                   StateManager stateManager, FileUpdateMonitor fileUpdateMonitor,
+                                   PreferencesService prefs) {
+        this.pivotEntry = pivotEntry;
+        this.config = config;
+        this.viewModel = new RelatedEntriesComponentViewModel(pivotEntry, repository, dialogService, databaseContext,
+                undoManager, stateManager, fileUpdateMonitor, prefs);
+        this.libraryTab = libraryTab;
+
+        initialize();
+        if (canLoadEntries()) {
+            viewModel.loadEntries();
+        } else {
+            relatedEntriesListView.setPlaceholder(buildLabel(Localization.lang("The selected entry doesn't have a DOI linked to it. Lookup a DOI and try again.")));
+        }
+
+        viewModel.relatedEntriesResultPropertyProperty().addListener((observable, oldValue, result) -> {
+            if (result.isPending()) {
+                showNodes(progressIndicator, cancelButton);
+                hideNodes(refreshButton, importEntriesButton);
+                relatedEntriesListView.getItems().clear();
+                relatedEntriesListView.setPlaceholder(buildLabel(Localization.lang("Loading...")));
+            } else if (result.isSuccess()) {
+                hideNodes(progressIndicator, cancelButton);
+                showNodes(refreshButton, importEntriesButton);
+
+                List<BibEntry> entries = result.asSuccess().value();
+                if (entries.isEmpty()) {
+                    relatedEntriesListView.setPlaceholder(buildLabel(Localization.lang("No articles found")));
+                } else {
+                    relatedEntriesListView.setItems(
+                            FXCollections.observableArrayList(entries.stream().map(entry ->
+                                    new CitationRelationItem(entry, false)).collect(Collectors.toList()))
```

Comment:
`.toList()` instead of `.collect(Collectors.toList())`

Reply:
Done.